node/rpc/Cargo.toml

```diff
@@ -30,3 +29,4 @@ sp-keystore = { workspace = true, default-features = true }
 sp-runtime = { workspace = true, default-features = true }
 substrate-frame-rpc-system = { workspace = true, default-features = true }
 substrate-state-trie-migration-rpc = { workspace = true, default-features = true }
+txpool-api = { package = "sc-transaction-pool-api", git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.42" }
```

Comment:
The idea behind workspace dependencies is that they're specified in root `Cargo.toml` and then used here with:
```
txpool-api = { workspace = true }
```
Meaning that we inherit the crate from the workspace (including version and all attributes)